lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
+
+  attr_reader :country,
+              :players
+
+  def initialize(country)
+    @country = country
+    @players = []
+
+  end
+
+  def eliminated?
```

Comment:
This may be better as an instance variable with a default value of `false`.  That way, when you get to later iterations, you can reset the value to true (after Croatia loses 😢 )